### PR TITLE
Add Exchange Rate

### PR DESCRIPTION
**Pull request by johnwatson484**

### CODE REVIEW

File: app/data/models/payment-request.js
Comment (Kirsten1822) [inline app/data/models/payment-request.js:1]: Just wondering, but do the payment request models need to be in any particular order?

Comment (johnwatson484) [reply app/data/models/payment-request.js:1]: They're currently grouped into related properties, eg customer references like frn, sbi, vendor etc are grouped together, debt data properties are together etc.  Then at the end the virtual properties added.

So there is some logic to it.

File: test/integration/local/payment-request/process-payment-request.test.js [hunk 1]
```diff
@@ -96,19 +96,20 @@ describe('process payment requests', () => {
...
+      x.value === 25000 &&
+      x.schemeCode === '80001' &&
+      x.accountCode === 'SOS273' &&
+      x.fundCode === 'DRD10')
+
+    const penaltyInvoiceLine = invoiceLinesRows.find(x => x.description === 'P02 - Over declaration penalty' &&
+      x.value === -10000 &&
+      x.schemeCode === '80001' &&
+      x.accountCode === 'SOS273' &&
+      x.fundCode === 'DRD10')
```
Comment (Kirsten1822) [inline test/integration/local/payment-request/process-payment-request.test.js:109]: Do these not need the agreementNumber anymore? Also is there anyway they could match the order of the mock invoice line defined  in the describe?

File: app/data/models/payment-request.js
Comment (Kirsten1822) [reply app/data/models/payment-request.js:1]: Okay thanks!

File: test/integration/local/payment-request/process-payment-request.test.js (hunk 1 above)
Comment (johnwatson484) [reply test/integration/local/payment-request/process-payment-request.test.js]: Yes it does.  Good spot.  Added agreement and reordered as suggested

**Review by Kirsten1822 — APPROVED** [review]
Happy to approve :)

**Pull request merged**